FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageJobQueueClient.cs

```diff
@@ -377,6 +377,8 @@ public async Task<IEnumerable<JobInfo>> GetJobsByIdsAsync(byte queueType, long[]
 
             await Task.WhenAll(tasks);
 
+            throttler.Dispose();
```

Comment:
Maybe just using `using var throttler` instead in line #363?

Reply:
Thanks, updated.